modules/nav.js

```diff
@@ -0,0 +1,56 @@
+const list = document.getElementById('list');
+const addNew = document.getElementById('addNew');
+const contact = document.getElementById('contact');
+export const logo = document.getElementById('logo');
+const addSection = document.getElementById('addSection');
+const listSection = document.getElementById('listSection');
+const contactSection = document.getElementById('contactSection');
+const home = document.getElementById('homePage');
+
+export class Navigation {
+  constructor() {
+    this.addSection = document.getElementById('addSection');
+    this.listSection = document.getElementById('listSection');
+    this.contactSection = document.getElementById('contactSection');
+    this.home = document.getElementById('homePage');
+  }
+
+  homePage() {
```

Comment:
- [ ] Kindly change this to an arrow function, as it makes your code look more conventional as it is part of the ES6 syntax